### PR TITLE
Fix routing fallback location

### LINKED ISSUE #2375: Refresh of admin edit pages fails
If you navigate to one of the edit pages in the admin site and do a browser refresh an HTTP 500 page is returned.  This is probably related to the routing changes with the webpack PR.  It also happens on the pool candidates in a pool page.  Maybe it's the UUID in the url?

### PR DESCRIPTION
**Pull request by petertgiles**
This branch changes the routing fallback file to an absolute path to avoid hitting the 10 rewrite limit in Apache on URLs with many segments.

~Note: leaving as draft until I can test on the dev server.~

Closes #2375

### CODE REVIEW

**Comment by tristan-orourke** [comment]
I just tested this on the dev server, it seems to work! Try going directly to https://dev-talentcloud.tbs-sct.gc.ca/en/admin/users/create

File: frontend/admin/src/.htaccess [hunk 1]
```diff
@@ -26,10 +26,6 @@ FallbackResource index.html
     # Send Requests To Front Controller...
     RewriteCond %{REQUEST_FILENAME} !-d
     RewriteCond %{REQUEST_FILENAME} !-f
```
Comment (tristan-orourke) [inline frontend/admin/src/.htaccess:28]: Since we removed the RewriteRule these conditions apply to, we might as well delete these as well

**Review by tristan-orourke — APPROVED** [review]
I saw this working on the dev server, I'm happy approving it.

File: frontend/admin/src/.htaccess (hunk 1 above)
Comment (petertgiles) [reply frontend/admin/src/.htaccess]: Thanks, I had no idea what they were for. :grin:

**Pull request merged**